network/common/src/main/java/org/apache/spark/network/TransportContext.java

```diff
@@ -97,6 +97,12 @@ public TransportServer createServer(int port, List<TransportServerBootstrap> boo
     return new TransportServer(this, port, rpcHandler, bootstraps);
```

Comment:
If you just call the overloaded `createServer` here, you could just modify the existing constructor in `TransportServer` instead of adding a new one.